ext/background/background.js

```diff
@@ -155,6 +155,11 @@ chrome.runtime.onMessage.addListener((request, _sender, sendResponse) => {
   return true;
 });
 
+// Respond to message from Range app to confirm extension is installed.
+chrome.runtime.onMessageExternal.addListener(function (request, sender, sendResponse) {
```

Comment:
Style nit: I think it's good to replace unused params with an underscore, or if there's multiple, prepend them with an underscore. See `background.js L28`